### ISSUE TITLE
How to determine installed version 

### ISSUE THREAD

**Issue by cb-davidelliott** [MicrosoftDocs/PowerShell-Docs#7024]
It would be helpful, on pages that you refer to versions of a product/feature, if you also included instructions on how to determine the currently installed versions I currently have, more as a sanity check and to give me peace of mind. I see the matrix you provide, and see things are supposed to be installed, however, I'm on a company owned PC and not sure what they may have done to it. So, it would be great if I could run a command and confirm that is the truth for my installation. 

This is an across the board type request, not just for this page, for any doc that references a product / capability, include instructions on how to determine my installed version(s) of that product / capability.

Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 680d1e35-ff25-ffb6-ca85-4b509d19d7e1
* Version Independent ID: 31dbba0a-b7d6-349b-2005-349eb5ae213e
* Content: [Windows Management Framework (WMF) - PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/windows-powershell/wmf/overview?view=powershell-7.1)
* Content Source: [reference/docs-conceptual/windows-powershell/wmf/overview.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/windows-powershell/wmf/overview.md)
* Product: **powershell**
* Technology: **powershell-conceptual**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

**Comment by sdwheeler** [comment]
WMF is not a single package. Windows does not provide a direct way to query the version. Specific versions are built into different versions of Windows. The table in the article covers that. If you have Windows 10 then you have WMF 5.1 

If you have an older version Windows then you have the version that shipped with that version of Windows. If WMF has been updated then your patch history will show the patch that update WMF. The patch number varies by OS version and WMF version. See the release notes linked in the article for the details.

If you are not on WMF 5.1 you need to be.

**Comment by cb-davidelliott** [comment]
I think you're missing the broader point I'm trying to communicate - granted not specific to your product.  If versions are important, which they must be because this page is all about versions, and the way I got to this page was from another page telling me to make sure the pre-req version of WMF 5.1 was installed. Then there should be an easy way for me to confirm the version, of whatever the product is, that is actually on my PC, EASILY, not jumping thru hoops, and reading release notes. You prove my point by saying WMF 5.1 is not a single package, okay, then Windows/Microsoft has that knowledge and shouldn't push it down to me to scratch my head and hope I've got right. There should be a simple way to confirm that yes, version X of product Y is, in fact, on my PC, and I know because the actual OS told me so, not a potentially out of date read me file. 

Thanks!
